x-pack/plugins/security_solution/common/api/detection_engine/rule_management/import_rules/rule_to_import.ts

```diff
@@ -29,14 +30,23 @@ export type RuleToImportInput = z.input<typeof RuleToImport>;
 export const RuleToImport = BaseCreateProps.and(TypeSpecificCreateProps).and(
   ResponseFields.partial().extend({
     rule_id: RuleSignatureId,
-    immutable: z.literal(false).default(false),
+    immutable: z.boolean().optional().default(false),
```

Comment:
31fc00c955a343659e703aad551ab0a51da7edee

Reply:
@jpdjere this actually causes an issue since we pass this type to `createRule`, which expects an `immutable` property on the rule. As a solution I [removed](75b0e3e4011ee418d4cad19d465a1eafd377535f) the `immutable` requirement on `createRule`, which I think is correct, but noting it here for review.